### ISSUE TITLE
Test table titles 

### ISSUE THREAD

**Issue by Maroyafaied** [DOI-ONRR/onrr.gov-site#2373]

**Comment by cthomasONRR** [comment]
Currently-

- In CMS, there’s no visual indicator to know if the "with headings" or "without headings" functionality is on/working. (Previously, using “with headings”, the heading text was bolded.) I asked that this visual indicator is put back.
- Title is the same format as H2, and we want to change it to H5 formatting.
- Title functionality only works for new tables, not existing tables. We need this functionality to work on existing tables.

![image](https://user-images.githubusercontent.com/72526208/209391758-46b4efdf-d22a-43fc-93ad-36c871d91c90.png)

**Comment by R-mohansai** [comment]
@cthomasONRR I have completed the functionality for Table headings for existing tables as well. Please review and comment it.

**Comment by cthomasONRR** [comment]
@R-mohansai @Maroyafaied 

When I'm testing creating a new table I'm seeing the following issues:
- When I go to set up a new table, I can add in the table title. But now, when I click away from the title text area (for example, to change the text from left-align to center-align, or to add rows or columns), the title text I added disappears. I can't show it in a screen grab, but can show on screen share. It seem really buggy.
- Title text is still the same format as H2, and we want to change it to H5 formatting.
- In CMS, there’s no visual indicator to know if the "with headings" or "without headings" functionality is on/working. (Previously, using “with headings”, the heading text was bolded.) I asked that this visual indicator is put back.

When I'm testing adding a title on existing tables:
- I can add the table title, but the title text is still the same format as H2. We want to change it to H5 formatting.
- When I add the table title in the CMS, save, and go back to it, that title text is missing from the CMS. This is something else I can get in a screen grab will will show when screen sharing.
- The title text is not spanning all the columns like it's displayed in the CMS. In preview/dev, this is what it looks like (title text is highlighted in yellow):
![image](https://user-images.githubusercontent.com/72526208/210382678-c4a8f48d-9a2d-4502-8976-23207c09d8e8.png)
- In CMS, there’s no visual indicator to know if the "with headings" or "without headings" functionality is on/working. (Previously, using “with headings”, the heading text was bolded.) I asked that this visual indicator is put back.

**Comment by cthomasONRR** [comment]
Table titles ("Captions" in the new table editor) are now working for non-collection tables without issue thanks to the new WYSIWYG editor!

I will make a new issue for collection-specific tables.

<img width="832" height="450" alt="Image" src="https://github.com/user-attachments/assets/9c53b949-7b4d-4edd-87a4-17681d11aa9b" />
